libs/component-library/src/lib/utils/animations.ts

```diff
@@ -0,0 +1,8 @@
+import { LayoutAnimation } from 'react-native';
```

Comment:
Might not be a problem in the latest rn version but I have found LayoutAnimation to quite buggy on Android in the past. 

(I do love how simple it is)

Reply:
It seems to be working okay for this use-case. I want to look at Reanimated layout transitions, but am waiting for a fix there for Android first.